### DISCUSSION TITLE
speechbrain/lang-id-commonlanguage_ecapa training results

### DISCUSSION THREAD

**Discussion by frankiedrake** [speechbrain/speechbrain#1682] [General]
First, sorry for duplicating the question here, but maybe this is more appropriate place for such a thing. 
I've just trained the model with data as specified in the recipe, but leaving only two languages from a provided dataset. The training process seemed finished without errors, but I'm not sure where to get the new generated model or, maybe I'm missing something.
As a training result I've got some checkpoint folders, e.g. `results/ECAPA-TDNN/1986/save/CKPT+2022-11-02+15-57-37+00` that contains some files, including `embedding_model.ckpt` which has the same size and checksum as the source one (which is used by default in the [recipe](https://huggingface.co/speechbrain/lang-id-commonlanguage_ecapa). Is this a newly generated model or this is something wrong in what I'm doing?

**Comment by Adel-Moumen** [comment]
Hello, 

Yes, the new model is stored in the CKPT. You can directly see this tutorial to know how to use it: https://colab.research.google.com/drive/1VH7U0oP3CZsUNtChJT2ewbV_q1QX8xre?usp=sharing

**Reply by frankiedrake** [reply]
> Hello,
> 
> Yes, the new model is stored in the CKPT. You can directly see this tutorial to know how to use it: https://colab.research.google.com/drive/1VH7U0oP3CZsUNtChJT2ewbV_q1QX8xre?usp=sharing

Thank you for the reply, but I just noticed that the file `embedding_model.ckpt` in result folder is the symlink to the original model. Where I can get the new model, generated by my own input data?

P.S. I learned the colab you've shared, before, but can't figure out what I need, sorry for my stupidness...

**Reply by frankiedrake** [reply]
This is what I have in a checkpoint folder
![image](https://user-images.githubusercontent.com/12988773/199700984-5f35de88-48bf-4b50-8136-0c012a119a76.png)

**Reply by frankiedrake** [reply]
If `embedding_model.ckpt` here is a resulting model, is it expected that it has the same size as the original one (I've used only 2 langs out from original model to train)

**Reply by frankiedrake** [reply]
I got it, sorry.

**Reply by Adel-Moumen** [reply]
Did the tutorial help you to better understand how to do this?

**Reply by frankiedrake** [reply]
> Did the tutorial help you to better understand how to do this?

I read the tutorial  before I started training the model it was just my misunderstanding of the process (I'm a beginner and I thought that the the model trained on a smaller amount of data will result in a smaller size of a resulting model file 😁)
